src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/price/MoneyTest.kt

```diff
@@ -5,9 +5,28 @@ import org.junit.jupiter.api.Test
 
 internal class MoneyTest {
   @Test
-  internal fun `multiplier override`() {
+  internal fun `multiplier operator override`() {
     assertThat(Money.valueOf(10.00) * 1.2).isEqualTo(Money.valueOf(12.00))
     assertThat(Money.valueOf(10.00) * 1.5).isEqualTo(Money.valueOf(15.00))
     assertThat(Money.valueOf(12.00) * 2.0).isEqualTo(Money.valueOf(24.00))
   }
+
+  @Test
```

Comment:
Hopefully these following two tests demonstrate the round is now behaving as expected.